tracing/scope.go

```diff
@@ -0,0 +1,66 @@
+// Trace a code block as a scoped span.
+// * Use instead of manual instrumentation: `tracer.Start()`/`span.End()`.
+// * Must call `InitTracing()` first.
+// * Automates start/end of span.
+// * Default span name is fully qualified function name.
+// * Tags file and line number where span started.
+// * If function returned error:
+//   * Span is tagged as error.
+//   * Sets span attributes `otel.status_code` and `otel.status_description`
+//     with error details.
+//   * Logs error details to span.
+
+package tracing
+
+import (
+	"context"
+	"runtime"
+	"strconv"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
+)
+
+type ScopeAction func(ctx context.Context) error
+
+// Call action function within a scoped tracing span.
```

Comment:
nit: docstring should begins with the name of the element it describes
https://go.dev/blog/godoc